services/core-api/app/api/utils/feature_flag.py

```diff
@@ -9,6 +9,8 @@ class Feature(Enum):
     MAJOR_PROJECT_REPLACE_FILE='major_project_replace_file'
     MINE_APPLICATION_FILE_UDPATE_ALERTS='mine_application_file_update_alerts'
     TRACTION_VERIFIABLE_CREDENTIALS='verifiable_credentials'
+    #if enabled the credential offer will be the current development state of all the 2.0 changes, Q1 2024
+    VC_MINES_ACT_PERMIT_20='vc_mines_act_permit_20'
```

Comment:
@taraepp This is the new flag i'm expecting. any issues with that name?

Reply:
Created! It's enabled in dev and disabled on prod. 👍